recipe/conda_forge_ci_setup/upload_or_check_non_existence.py

```diff
@@ -198,7 +206,7 @@ def upload_or_check(
                             "re-uploading.".format(path, owner)
                         )
                         delete_dist(token_fn, path, owner, channel)
-                        upload(token_fn, path, owner, channel)
+                        upload(token_fn, path, owner, channel, private_upload)
```

Comment:
You won't ever need to pass this in the verify blocks so we should remove it here.

Reply:
I don't understand why in `validate` block we call `upload`? are the package removed after? if it does, where?

Reply:
```python
                        delete_dist(token_fn, path, owner, channel)
                        upload(token_fn, path, owner, channel)
```

Reply:
this is to another channel and works already